src/__main__.py

```diff
@@ -5,13 +5,16 @@
 from ui import Gui
 
 
+
 def main():
     '''Entry point for the application.'''
 
     config = Config()
 
     if not os.path.exists(config.DOWNLOADS_FOLDER):
         os.makedirs(config.DOWNLOADS_FOLDER, exist_ok=True)
+    if not os.path.exists(config.DOCUMENTS_FOLDER):
+        os.makedirs(config.DOWNLOADS_FOLDER, exist_ok=True)
```

Comment:
Should this be `os.makedirs(config.DOCUMENTS_FOLDER, exist_ok=True)`?

Reply:
Good catch; I've just changed it